openlibrary/templates/site/alert.html

```diff
@@ -1,6 +1,9 @@
 $if not is_bot():
   $:get_donation_include()
 
+$ lang = get_lang() or 'en'
+$ use_lang = lang if lang in get_supported_languages() else 'en'
```

Comment:
Something like this should do the trick! This will (1) avoid calling gets_supported_languages() twice which is a small perf boost, and (2) 

```suggestion
$ supported_languages = get_supported_languages()
$ active_ui_lang = supported_languages.get(lang) or supported_language.get('en')
```

Then you can do:

```html
<span>$active_ui_lang['localized'] ($active_ui_lang['code'])</span>
```

This will require duplicating the `code` also inside the dictionary, but I think that's a worthwhile change!